includes/api/save-abandonment-data.php

```diff
@@ -80,10 +94,10 @@ function saveCartAbandonmentData(WP_REST_Request $request)
         //Insert data for tracking started checkout.
         $eventData = wck_build_cart_data($cart);
         if (empty($eventData['$extra']['Items'])) {
-            $response['status']  = false;
-            $response['message'] = 'cart item not exist in kalviyo';
-            $statusCode          = 400;
-            return new WP_REST_Response($response, $statusCode);
+            $response['status']    = false;
+            $response['message']   = 'cart item does not exist in klaviyo';
```

Comment:
Are we using  $response['message'] somewhere?

Reply:
Removing it and updating reason in the result['response']

Reply:
Appending to result['response']